src/MakersOfDenmark.Domain/Models/User/MODRole.cs

```diff
@@ -0,0 +1,16 @@
+﻿using MakersOfDenmark.Domain.Enums;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace MakersOfDenmark.Domain.Models.User
+{
+    public class MODRole : Entity<int>
+    {
+        public MODUser User { get; set; }
+        public MakerSpace MakerSpace { get; set; }
+        public MakerSpaceRoles  Role { get; set; }
+        public DateTimeOffset CreatedAt { get; set; } = DateTimeOffset.UtcNow;
```

Comment:
I samme stil med min kommentar på MakerSpace.cs (om vi skulle lave Request/Response), så bør vi vel ikke have properties som CreatedAt mm. i domænet?

Reply:
Jeg kan se fordele og ulemper ved ikke at have en createdat i domænet.

Det kommer an på use casen. Skal vi bruge CreatedAt udelukkende til database kald? Eller skal vi bruge den i vores domæne logik?

Hvis det første så synes jeg det burde være en Shadow Property som EF Core generere for os. Hvis det sidste, så burde den være i domænet, men under Entity<T>.